Packs/Cisco-umbrella/Integrations/Cisco-umbrella-investigate/Cisco-umbrella-investigate.py

```diff
@@ -748,113 +748,115 @@ def get_domain_command():
             }
-            dbotscore = 3
-
-        context[outputPaths['dbotscore']] = {
-            'Indicator': domain,
-            'Type': 'domain',
-            'Vendor': 'Cisco Umbrella Investigate',
-            'Score': dbotscore,
-            'Reliability': reliability
-        }
 
-        contents.append({
-            'Risk Score': risk_score,
-            'Secure Rank': secure_rank,
-            'Populairty': popularity,
-            'Demisto Reputation': scoreToReputation(dbotscore),
-            'First Queried time': first_queried,
-        })
-
-        # Domain reputation + [whois -> whois nameservers -> whois emails] + domain categorization
-        results.append({
-            'Type': entryTypes['note'],
-            'ContentsFormat': formats['json'],
-            'Contents': [contents, whois, name_servers, emails, domain_categorization_table],
-            'ReadableContentsFormat': formats['markdown'],
-            'HumanReadable':
-                tableToMarkdown('"Umbrella Investigate" Domain Reputation for: ' + domain, contents, headers)
-                + tableToMarkdown('"Umbrella Investigate" WHOIS Record Data for: ' + domain, whois, headers,
-                                  date_fields=["Last Retrieved"])
-                + tableToMarkdown('Name Servers:', {'Name Servers': name_servers}, headers)
-                + tableToMarkdown('Emails:', emails, ['Emails'])
-                + tableToMarkdown('Domain Categorization:', domain_categorization_table, headers),
-            'EntryContext': context
-        })
+            contents.append({
+                'Risk Score': risk_score,
+                'Secure Rank': secure_rank,
+                'Populairty': popularity,
+                'Demisto Reputation': scoreToReputation(dbotscore),
+                'First Queried time': first_queried,
+            })
 
+            # Domain reputation + [whois -> whois nameservers -> whois emails] + domain categorization
+            results.append({
+                'Type': entryTypes['note'],
+                'ContentsFormat': formats['json'],
+                'Contents': [contents, whois, name_servers, emails, domain_categorization_table],
+                'ReadableContentsFormat': formats['markdown'],
+                'HumanReadable':
+                    tableToMarkdown('"Umbrella Investigate" Domain Reputation for: ' + domain, contents, headers)
+                    + tableToMarkdown('"Umbrella Investigate" WHOIS Record Data for: ' + domain, whois, headers,
+                                      date_fields=["Last Retrieved"])
+                    + tableToMarkdown('Name Servers:', {'Name Servers': name_servers}, headers)
+                    + tableToMarkdown('Emails:', emails, ['Emails'])
+                    + tableToMarkdown('Domain Categorization:', domain_categorization_table, headers),
+                'EntryContext': context
+            })
+        except Exception:
+            pass
```

Comment:
Let's distinguish a `not found error` with other exceptions so we'd only skip failure for domains that this vendor does not have information on.
Also, instead of just `pass`, let's add a text so customers would have information and output about the failed domain. 

e.g
![Screen Shot 2021-12-01 at 10 49 23](https://user-images.githubusercontent.com/21142167/144201862-62929742-a2bc-4993-a8e4-b3c45ecf356c.png)

Reply:
Right, I changed the Exception type. However about giving information about the exception, the information is already logged by the function that raised the exception.

Reply:
In addition to the logs, I want us to add an actual output regarding the domain as can be seen in the image above.

Reply:
<img width="839" alt="Screen Shot 2021-12-01 at 17 22 05" src="https://user-images.githubusercontent.com/79846533/144262006-0b6e0e79-4c5b-40c5-b43b-bccc89f4a61f.png">
Added